src/tests/api/test_subevents.py

```diff
@@ -50,6 +109,242 @@ def test_subevent_list(token_client, organizer, event, subevent):
     assert [] == resp.data['results']
 
 
+@pytest.mark.django_db
+def test_subevent_get(token_client, organizer, event, subevent):
+    res = dict(TEST_SUBEVENT_RES)
+    res["id"] = subevent.pk
+    resp = token_client.get('/api/v1/organizers/{}/events/{}/subevents/{}/'.format(organizer.slug, event.slug,
+                                                                                   subevent.pk))
+    assert resp.status_code == 200
+    assert res == resp.data
+
+
+@pytest.mark.django_db
+def test_subevent_create(token_client, organizer, event, subevent, meta_prop, item):
+    resp = token_client.post(
+        '/api/v1/organizers/{}/events/{}/subevents/'.format(organizer.slug, event.slug),
+        {
+            "name": {
+                "de": "Demo Subevent 2020 Test",
+                "en": "Demo Subevent 2020 Test"
+            },
+            "active": False,
+            "date_from": "2017-12-27T10:00:00Z",
+            "date_to": "2017-12-28T10:00:00Z",
+            "date_admission": None,
+            "presale_start": None,
+            "presale_end": None,
+            "location": None,
+            "item_price_overrides": [],
+            "variation_price_overrides": [],
+            "meta_data": {
+                "type": "Workshop"
+            },
+        },
+        format='json'
+    )
+    assert resp.status_code == 201
+    assert not organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).active
+    assert organizer.events.get(slug="dummy").subevents.get(id=resp.data['id']).meta_values.filter(
```

Comment:
Just a style thing, you can just use the ``event`` fixture variable, no need to get it from the database. Not a real problem, though ;)